mease_lab_to_nwb/convert_syntalos/syntaloseventinterface.py

```diff
@@ -34,7 +35,7 @@ def run_conversion(self, nwbfile: NWBFile, metadata: dict):
         event_file = self.source_data['file_path']
         events_data = pd.read_csv(event_file, header=0)
         split_first_col = [x.split(";") for x in events_data['Time;Tag;Description']]
-        event_timestamps = [int(x[0]) for x in split_first_col]
+        event_timestamps = [int(x[0]) / 1E3 for x in split_first_col]
```

Comment:
@CodyCBakerPhD a nicer way to do this is:

```
events_data = pd.read_csv(event_file, delimiter=';')
event_timestamps = events_data["Time"].to_numpy() / 1E3
event_labels = events_data["Tag"].to_numpy()
```

Reply:
Cleaned it up now, thanks for pointing out the delimiter usage here